ansible/roles/stack-sunbird/templates/dhiti-service.env

```diff
@@ -0,0 +1,85 @@
+TELEMETRY_DATASOURCE_NAME={{dhiti_telemetry_datasource_name | default("dev_sl_telemetry")}}
+EVIDENCE_DATASOURCE_NAME={{dhiti_evidence_datasource_name | default("dev_sl_observation_evidences")}}
+SURVEY_DATASOURCE_NAME={{dhiti_survey_datasource_name | default("dev_sl_survey_druid")}}
+SURVEY_EVIDENCE_DATASOURCE_NAME={{dhiti_survey_evidence_datasource_name | default("dev_sl_survey_evidences")}}
+OBSERVATION_EVIDENCE_DATASOURCE_NAME={{dhiti_observation_evidence_datasource_name | default("dev_sl_observation_evidences")}}
+CONTENT_REPORT_THRESHOLD={{dhiti_content_report_threshold | default("5")}}
+ENTITY_SCORE_REPORT_THRESHOLD={{dhiti_entity_score_report_threshold | default("5")}}
+OBSERVATION_SCORE_REPORT_THRESHOLD={{dhiti_observation_score_report_threshold | default("2")}}
+ASSESSMENT_SUBMISSION_REPORT_THRESHOLD={{dhiti_assessment_submission_report_threshold | default("3")}}
+EVIDENCE_THRESHOLD={{dhiti_evidence_threshold | default("3")}}
+
+#Auth Config
+sunbird_keycloak_auth_server_url={{keycloak_auth_server_url}}
+sunbird_keycloak_realm={{keycloak_realm | default("sunbird")}}
+sunbird_keycloak_client_id={{sunbird_keycloak_client_id | default("portal")}}
+sunbird_keycloak_public={{sunbird_keycloak_public | default("true")}}
+sunbird_cache_store={{sunbird_cache_store | default("")}}
+sunbird_cache_ttl={{dhiti_sunbird_cache_ttl | default("1800")}}
+
+URL_PREFIX={{dhiti_url_prefix | default("api/v1")}}
+
+#ASSESSMENT service
+ASSESSMENT_SERVICE_APPLICATION_ENDPOINT=http://sl-assessments-service:4201.{{namespace}}.svc.cluster.local
+ASSESSMENT_SERVICE_BASE_URL={{dhiti_assessment_service_base_url | default("/assessment/")}}
+
+#KENDRA service
+KENDRA_APPLICATION_ENDPOINT=http://kendra-service:4202.{{namespace}}.svc.cluster.local
+KENDRA_BASE_URL={{dhiti_kendra_base_url | default("/kendra/")}}
+
+#Gotenberg configuration
+GOTENBERG_URL={{dhiti_gotenberg_url | default ("http://gotenberg-service:3000/convert/html")}}
+
+#Highchart configuration
+HIGHCHART_URL=http://highcharts-service:8080.{{namespace}}.svc.cluster.local
+
+# FILE PATHS FOR PORTAL REPORTS
+USER_NEVER_LOGGED_IN={{dhiti_user_never_logged_in}}
+UNIQUE_ACTIVE_USERS={{dhiti_unique_active_users}}
+TOP_SCORE_QUIZ={{dhiti_top_score_quiz}}
+CONTENT_RATINGS={{dhiti_content_ratings}}
+MAP_DATA_RESOURCES={{dhiti_map_data_resources}}
+LOGIN_TREND={{dhiti_login_trend}}
+LOGIN_PERCENTAGE={{dhiti_login_percentage}}
+LEARNING_TOP_SCORE_QUIZ={{dhiti_learning_top_score_quiz}}
+PARTICIPATION_PERCENTAGE={{dhiti_participation_percentage}}
+LAST_UPDATED_DATE={{dhiti_last_updated_date}}
+DAILY_AVERAGE_GROWTH={{dhiti_daily_average_growth}}
+COUNT_CONTENT_RATING={{dhiti_count_content_rating}}
+AVERAGE_TIME_SPENT={{dhiti_average_time_spent}}
+AVERAGE_RATING_CONTENT={{dhiti_average_rating_content}}
+APP_COUNT={{dhiti_app_count}}
+ADOPTION_CONTENT={{dhiti_adoption_content}}
+MULTI_RESOURCE={{dhiti_multi_resource}}
+MULTI_SELECTION={{dhiti_multi_selection}}
```

Comment:
Hi, can we have one-liner comments denoting what these variables are? that'll be helpful if you want to fill out a variable, and don't want to skim through the entire documentation.

Reply:
Sure @rjshrjndrn